### PR TITLE
travis: run nosetests-3.4

### LINKED ISSUE #6097: Travis tests are not running
Check with Travis and ensure following:
- Smoketests should run
- UI tests should run
- Lint checks if any should run

... and that the build MUST fail if any of the above fail.

For example, right now smoketests aren't running - https://app.travis-ci.com/github/apache/cloudstack/builds/247736668

Pl check and fix this for both 4.16 and 4.17/main as applicable.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
CI/CD
~~~

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes #6097 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: tools/travis/script.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ set -e
 
 for suite in "${TESTS[@]}" ; do
   echo "Currently running test: $suite"
-  time nosetests --with-xunit --xunit-file=integration-test-results/$suite.xml --with-marvin --marvin-config=setup/dev/advanced.cfg test/integration/$suite.py -s -a tags=advanced,required_hardware=false --zone=Sandbox-simulator --hypervisor=simulator || true ;
+  time nosetests-2.7 --with-xunit --xunit-file=integration-test-results/$suite.xml --with-marvin --marvin-config=setup/dev/advanced.cfg test/integration/$suite.py -s -a tags=advanced,required_hardware=false --zone=Sandbox-simulator --hypervisor=simulator || true ;
```
Comment (rohityadavcloud) [inline tools/travis/script.sh:33]: @weizhouapache should this be nosetests-3.4? We moved to python3, so we should ensure to install dependencies via pip3 and run tests using nosetests-3.4?

Comment (weizhouapache) [reply tools/travis/script.sh]: @rohityadavcloud 
yes, the travis test still failed. it should be nosetest-3.4

**Review by sureshanaparti — APPROVED** [review]

**Comment by nvazquez** [comment]
Thanks @weizhouapache - had closed/opened to kick Travis again

**Review by nvazquez — APPROVED** [review]
LGTM - verified jobs are running again, thanks @weizhouapache

**Pull request merged**